packages/devp2p/examples/peer-communication.cts

```diff
@@ -7,10 +7,10 @@ import chalk from 'chalk'
 import type LRUCache from 'lru-cache'
 
 const LRU = require('lru-cache')
```

Comment:
Can't LRU be imported with ESM syntax?

Reply:
Their docs seem to indicate that the import syntax should work: http://isaacs.github.io/node-lru-cache/

Reply:
Doesn't seem to work in this instance.  I think since I've changed these scripts to be `cts` (ie explicitly CommonJS), we have to continue to require it.  I'm sure there's a way to do it with import but the exports in the version of LRUCache we're using here don't play nicely with Typescript's `import ...` resolution.